migration/src/m0000641_update_product_status.rs

```diff
@@ -0,0 +1,130 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+#[allow(deprecated)]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(ProductStatus::Table)
+                    .add_column(ColumnDef::new(ProductStatus::Package).string())
+                    .to_owned(),
+            )
+            .await?;
+
+        // Transform data: populate `package` from `base_purl`
```

Comment:
Thanks for going to the effort -- seems a tad gnarly. I confirmed the migration of my local data worked fine.

Reply:
It is :/ Thanks for checking it